src/components/group-dropdown.tsx

```diff
@@ -0,0 +1,63 @@
+import { Select, Title } from '@mantine/core';
+import classes from './componentStyles/patient-dropdown.module.css';
+import { useEffect, useState } from 'react';
+
+export interface GroupDropdownProps {
+  setGroupId: (id: string | null) => void;
+}
+
+/*
+ * Component is the selectable dropdown populated with the group data. Takes in a function as props
+ * to be able to set id in the query string.
+ */
+export default function GroupDropdown({ setGroupId }: GroupDropdownProps) {
+  const [data, setData] = useState<string[]>([]);
+  const [error, setError] = useState(false);
+  const [selectedId, setSelectedId] = useState<string | null>('');
+
+  useEffect(() => setGroupId(selectedId));
+
+  // try to change this to be SSG
+  useEffect(() => {
+    fetch('http://localhost:3000/Group')
```

Comment:
We're going to want to pull this url out - probably into an environment variable. Additionally, if we're generally going to assume that our server is running on port 3000, it would be more convenient to default start the application on port 3001, so that the default setup works together. (the app port could be specified in an environment variable as well, or we could just add `-p 3001` to the package.json start and dev scripts for now.